starter-code/package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "nodemon ./bin/www"
```

Comment:
Sneaky 😏 .

For the next time, try something like that:

```
"script": {
    "start": "node ./bin/www",
    "dev": "nodemon ./bin/www"
}
```

Now you can do `npm run dev` to start nodemon, and you don't have to change the start script ;)